src/CommunityToolkit/Xamarin.CommunityToolkit.UnitTests/Converters/BaseConverterOneWay_Tests.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using NUnit.Framework;
+using Xamarin.CommunityToolkit.Converters;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.UnitTests.Converters
+{
+	[TestOf(typeof(BaseConverterOneWay<string, Color>))]
+	public class BaseConverterOneWay_Tests
+	{
+		#region AllowsNullOrDefault = true
+
+		/// <summary>
+		/// Derivation of <see cref="BaseConverterOneWay{TFrom,TTo}"/> to test the new <see cref="AllowsNullOrDefault"/> handling, allowing null.
+		/// </summary>
+		private class ColorNameToColorConverter : BaseConverterOneWay<string, Color>
+		{
+			protected override bool AllowsNullOrDefault => true;
+
+			public override Color ConvertFrom(string? value) =>
+				value switch
+				{
+					// By setting AllowsNullOrDefault to true, it is possible to handle this case by the derived class.
+					null => Color.Black,
+					"Red" => Color.Red,
+					"Blue" => Color.Blue,
+					_ => throw new ArgumentException($"{value} unknown.")
+				};
+		}
```

Comment:
It would be nice to combine this with the other `ColorNameToColorConverterDisallowingNull` class and simply pass in a value for `AllowsNullOrDefault` to the constructor. Perhaps we could move it to the mocks and name it so it is clearly a mock?